content/docs/hooks-reference.md

```diff
@@ -314,7 +314,7 @@ function init(initialCount) {
   return {count: initialCount};
 }
 
-function reducer(state, action) {
+function reducer(state, action, init) {
```

Comment:
This param is not passed by React. The `init` referenced in `reducer` is declared  in https://github.com/reactjs/reactjs.org/pull/3568/files#diff-a1c8ec10dc40c86b4bbd55ad3f0126835bd45e9aa9e56c846a2e28ad6dd6acffR313.

Does this clear things up?

Reply:
Yes it does. Thank you